README.md

```diff
@@ -34,6 +33,15 @@ Commonly used Gulp tasks shared across projects.
 - `gulp fractal:start`: Start Fractal server (if config path exists in `package.json`)
 - `gulp fractal:build`: Build static version of Fractal project
 
+## Options for js/css tasks
+
+`--minify` | Render additional minified files, suffixed with `.min`. Will be `true` in production builds.
+`--sourcemaps` | Render sourcemaps.
+`--fail-after-error` | See the 'Continuous Integration' section for details.
+`--no-concat` | By default, assets are compiled into a single file. Use this flag to keep files invididual.
+
+You can pass an `options` object inside `gulpPaths.scripts`/`gulpPaths.styles` in your `package.json` to set options for each respective task.
```

Comment:
While I can see the benefit in order to be more flexible I wonder if this is actually practicable. Under which circumstances would you need to have the scripts concatenated but the styles?

Reply:
Actually, this is exactly the kind of behavior we need for bnn right now (which is why I included it):
Our JS files need to be concated, but we rely on the CSS not being concatenated (`fractal.css` is included in the styleguide).
Of course, we could refactor that, but I though it to be more flexible if we can pass any options. It does not really require a lot of additional code (just an additional argument passed to `Object.assign`)